### ISSUE TITLE
Camp Woodwind missing after Apocalypse (0.7.0.0) patch

### ISSUE THREAD

**Issue by AshKapow** [GlitchedMythos/se-widgets#62]
After the Apocalypse (0.7.0.0) patch, we are missing the Camp Woodwind map [Patch Notes](https://thenerdstash.com/phasmophobia-update-0-7-0-0-patch-notes/)

Suggestion, after line 112 in widget.js:
```
wo: "Camp Woodwind",
wood: "Camp Woodwind",
woodwind: "Camp Woodwind",
campwoodwind: "Camp Woodwind"
```

**Comment by billyjbryant** [comment]
This is resolved in #69

**Comment by GlitchedMythos** [comment]
Confirmed that I've seen this and looking for it in #69
